recipes/ticcltools/meta.yaml

```diff
@@ -0,0 +1,66 @@
+{% set name = "ticcltools" %}
+{% set version = "0.6" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://github.com/LanguageMachines/{{ name }}/releases/download/v{{ version }}/{{ name }}-{{ version }}.tar.gz
+  sha256: "10c243b2f45da86b4a40cb37af78b45bcd6756e5035d43af99fc47043711cdb0"
+
+build:
+  number: 0
+  skip: True  # [win]
+  run_exports:
+    - ticcltools
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - autoconf
+    - autoconf-archive
+    - automake
+    - libtool
+    - pkg-config
+  host:
+    - ticcutils
+  run:
+    - ticcutils
+
+test:
+  commands:
+    - test -d ${PREFIX}/include/ticcltools  # [unix]
+    - for fn in ${PREFIX}/bin/TICCL-{LDcalc,anahash,chain,indexer{,NT},lex{clean,stat},mergelex,rank,stats,unk}; do test -f ${fn}; done  # [unix]
+    - for fn in ${PREFIX}/bin/W2V-{analogy,dist,near}; do test -f ${fn}; done  # [unix]
+    - test -f ${PREFIX}/lib/libticcl.a  # [unix]
+
+about:
+  home: https://github.com/LanguageMachines/ticcltools
+  license: GPL-3.0
+  license_family: GPL
+  license_file: COPYING
+  summary: 'Tools for TICCL: Text Induced Corpus-Cleanup'
+  dev_url: https://github.com/LanguageMachines/ticcltools
+  description: |
+    TicclTools is a collection of programs to process datafiles, together they constitute the bulk of TICCL: Text Induced Corpus-Cleanup.
+    The main programs in this colection are:
+    * TICCL-indexer and TICCL-indexerNT:
+      a tool to create an exhaustive index to all lexical variants given a particular Levenshtein or edit distance in a corpus.
+    * TICCL-anahash:
+      a tool to create anagram hashes form a word frequency file. Also creates ab 'alphabet' file of the unicode characters that are present in the corpus.
+    * TICCL-LDcalc:
+      a proprocessing tool for TICCL-rank. Gathers the info from TICC-anahash, TICCL-indexer, TICCL-lexstat and TICCL-unk
+    * TICCL-rank:
+      ranks a word varian list according to al lot of criteria
+    * TICCL-unk:
+      a cleanup tool for word frequency lists. creates a 'clean' file with desirable words, an 'unk' file with uncorrectable words and a 'punct' file with words that would be clean after removing puncuation.
+    * TICCL-lexstat:
+      convert an 'alphabet' file (from TICCL-anahash) into a frequency list of hashes and optionally a list of confusions.
+
+extra:
+  recipe-maintainers:
+    - egpbos
+    - jvdzwaan
+    - martinreynaert
```

Comment:
@jvdzwaan and @martinreynaert should confirm they are OK as maintainers.

Reply:
I am OK with being regarded as a maintainer.

Martin Reynaert

Reply:
I'm willing to be a maintainer.